llvm/test/tools/UpdateTestChecks/update_analyze_test_checks/Inputs/loop-access-analysis.ll.expected

```diff
@@ -81,7 +81,7 @@ define void @test_brace_escapes(ptr noundef %arr) {
 ; CHECK-NEXT:      Grouped accesses:
 ; CHECK-NEXT:        Group [[GRP4]]:
 ; CHECK-NEXT:          (Low: {(64 + %arr),+,64}<%loop.1> High: {(8064 + %arr),+,64}<%loop.1>)
-; CHECK-NEXT:            Member: {{(64 + %arr),+,64}<%loop.1>,+,8}<%loop.2>
+; CHECK-NEXT:            Member: {{\{{}}(64 + %arr),+,64}<%loop.1>,+,8}<%loop.2>
```

Comment:
Is this the correct result? Shouldn't this be `{{\{\{}}`?

Reply:
It works for {{, but not for `{{{` or more. I updated the code to use a replacement function that escapes each { in the sequence; not sure if there's some python regex magic that does that without a separate function